app/resources/numismatics/accessions/numismatics/accession.rb

```diff
@@ -11,7 +11,7 @@ class Accession < Resource
     attribute :numismatic_citation, Valkyrie::Types::Array.of(Numismatics::Citation).meta(ordered: true)
 
     # descriptive metadata
-    attribute :accession_number, Valkyrie::Types::Integer
+    attribute :accession_number, Valkyrie::Types::Coercible::Integer.optional
```

Comment:
Question: Should we be using BetterParamsInteger for this and the other instances where we've replaced it with `Coercible::Integer.optional`?